### PR TITLE
Dev majkji

### PR DESCRIPTION
**Pull request by majkji**
Bitwise tests

### CODE REVIEW

**Comment by majkji** [comment]
@googlebot I signed it!

File: tests/bit_ops.rs [hunk 1]
```diff
@@ -0,0 +1,424 @@
+
```
Comment (kaidokert) [inline tests/bit_ops.rs:1]: You need to put the Apache license with Google copyright here at the top under Google CLA agreement. You can add a separate author: line below Copyright line if you want, see some examples at https://opensource.google/docs/copyright/#c

**Comment by kaidokert** [comment]
Please run rust-fmt to get the CI to pass.

You can also locally install `pre-commit` to automate checks locally before pushing the changes, some generic instructions at https://pre-commit.com/#installation

Separately, i'll look into as to why Coveralls didn't run properly

**Comment by kaidokert** [comment]
This looks good now - coverage still didn't complete but there's something wrong with the repo configuration. It should run 
fine after merge.

I'll go ahead and merge - thank you very much for this contribution !

**Pull request merged**